### PR TITLE
Sc 680 task add feemanager address during init of governor

### PR DESCRIPTION
**Pull request by FHieser**

### CODE REVIEW

File: script/deployment/DeploymentScript.s.sol [hunk 1]
```diff
@@ -253,21 +253,31 @@ contract DeploymentScript is Script {
         // Fetch the treasury address
         address treasury = vm.envAddress("TREASURY_ADDRESS");
 
+        console2.log(
+            "-----------------------------------------------------------------------------"
+        );
+
+        console2.log("Deploy Fee Manager \n");
+
+        feeManager = deployFeeManager.createProxy(communityMultisig); //@note owner of the FeeManagerBeacon will be the communityMultisig. Is that alright or should I change it to Governor? Needs more refactoring that way
```
Comment (Zitzak) [inline script/deployment/DeploymentScript.s.sol:262]: Sounds right to me

File: script/external/DeployGovernor_v1.s.sol [hunk 2]
```diff
@@ -43,14 +44,20 @@ contract DeployGovernor_v1 is Script {
...
-        return run(communityMultisig, teamMultisig, timelockPeriod);
+        return run(communityMultisig, teamMultisig, timelockPeriod, feeManager);
     }
 
     function run(
         address communityMultisig,
         address teamMultisig,
-        uint timelockPeriod
+        uint timelockPeriod,
+        address initialFeeManager
```
Comment (Zitzak) [inline script/external/DeployGovernor_v1.s.sol:60]: Why do we actually get the variable from our .env as well as from the function call? I'm seeing that this happens elsewhere also

File: src/external/governance/Governor_v1.sol [hunk 3]
```diff
@@ -174,11 +177,10 @@ contract Governor_v1 is ERC165, IGovernor_v1, AccessControlUpgradeable {
 
     /// @inheritdoc IGovernor_v1
     function setFeeManager(address newFeeManager)
-        external
+        public
```
Comment (Zitzak) [inline src/external/governance/Governor_v1.sol]: Why not keep it external?

Comment (Zitzak) [reply src/external/governance/Governor_v1.sol]: Is there a need for it to be called from within this contract?

**Review by Zitzak — APPROVED** [review]
Only some question, no change required

File: script/external/DeployGovernor_v1.s.sol (hunk 2 above)
Comment (FHieser) [reply script/external/DeployGovernor_v1.s.sol:60]: This is to test the scripts in the Make pre-commit
We basically mock the other addresses that way

File: src/external/governance/Governor_v1.sol (hunk 3 above)
Comment (FHieser) [reply src/external/governance/Governor_v1.sol]: I did that before the internal function
Will change it

File: script/external/DeployGovernor_v1.s.sol (hunk 2 above)
Comment (Zitzak) [reply script/external/DeployGovernor_v1.s.sol:60]: Ahh okay, makes sense

**Pull request merged**